src/runner/index.js

```diff
@@ -25,30 +25,36 @@ export default class Runner extends EventEmitter {
     }
 
     // Run task
-    async _runTask (reporterPlugin, browserSet, tests) {
+    _runTask (reporterPlugin, browserSet, tests) {
         var task     = new Task(tests, browserSet.connections, this.proxy, this.opts);
         var reporter = new Reporter(reporterPlugin, task, this.opts.reportOutStream);
 
         task.on('browser-job-done', job => browserSet.releaseConnection(job.browserConnection));
 
-        try {
-            await Promise.race([
+        var promise = Promise
+            .race([
```

Comment:
I'd prefer to keep async/await syntax for this. We need to refactor this part to separate function with some meaningful name then.